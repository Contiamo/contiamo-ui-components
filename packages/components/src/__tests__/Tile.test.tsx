import * as React from "react"
import { render } from "enzyme"
import { Tile as ThemelessTile } from "../index"
import wrapDefaultTheme from "../utils/wrap-default-theme"

const Tile = wrapDefaultTheme(ThemelessTile)

describe("InfoTile", () => {
  it("Should render", () => {
    const renderedComponent = render(<Tile label="Country">DE</Tile>)
    expect(renderedComponent).toMatchSnapshot()
  })
})
