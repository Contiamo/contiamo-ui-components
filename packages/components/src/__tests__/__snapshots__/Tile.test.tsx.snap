// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoTile Should render 1`] = `
<div
  class="css-1naufuj"
>
  <small
    class="css-1u50kn8"
  >
    Country
  </small>
  <div
    class="css-1x16d7v"
  >
    DE
  </div>
</div>
`;
